common/artifacts/908_wc_artifacts_relic.txt

```diff
@@ -88,16 +63,45 @@ chalice_of_trollbanes = {
 	stacking = no
 	indestructible = yes
 
-	health = 1
+	health = 0.7
 
-	monthly_character_prestige = 1
+	monthly_character_prestige = 0.8
 
 	light_group_opinion = 5
 
 	picture = "GFX_inv_misc_trophy_argent"
 	slot = relic
 }
 
+# Rare
+dark_crystal = {
+	quality = 5
```

Comment:
> Rare

> quality 5

Reply:
right. thx. moved it but forgot to change number. done